package.json

```diff
@@ -33,8 +33,9 @@
     "array.prototype.flat": "^1.2.1",
     "array.prototype.flatmap": "^1.2.1",
     "chalk": "^2.4.2",
+    "es6-promisify": "^6.0.1",
```

Comment:
why this, over https://npmjs.com/util.promisify, which is a polyfill for the built-in node functionality?